docs/02-app/02-api-reference/04-functions/use-params.mdx

```diff
@@ -18,7 +18,7 @@ export default function ExampleClientComponent() {
   // `params` -> { tag: 'shoes', item: 'nike-air-max-97' }
   console.log(params)
 
-  return <></>
+  return null
```

Comment:
Since it is an example component, it seems like the fragment was used to abstract the rest code, not just returning null.

How about something like this?

```suggestion
  return <>{/* ...rest */}</>
```

Reply:
My example was more about what happens when you don't return anything. I wanted to keep the functionality the same as for the previous docs.
In my real-life production code, we don't expect anything in the return statement anyway for this component, and the `<></>` react fragment is confusing for new developers.

But your change is good too!
Can we do it in the next pr?

Let me know what you think!

Reply:
```suggestion
  return '...'
```

Reply:
Let's go with this! Will further use`'...'` throughout the examples instead of `null` or `<></>` 😄